tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/ProjectUI.java

```diff
@@ -191,4 +200,24 @@ public void stop( BundleContext context ) throws Exception
         super.stop( context );
     }
 
+    private void addWorkspaceResourceListener()
+    {
+        ResourcesPlugin.getWorkspace().addResourceChangeListener( events -> {
+            boolean isRemoved = MigrationUtil.removeMigrationProblemsFromResource( events.getResource() );
+
+            if( isRemoved )
+            {
+                UIUtil.async( () -> {
+                    MigrationView mv = (MigrationView) UIUtil.showView( MigrationView.ID );
```

Comment:
What if view is not shown at all?  we shouldn't show the view if it wasn't already open.